pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/FilterOperator.java

```diff
@@ -56,8 +56,8 @@ public class FilterOperator extends MultiStageOperator {
   private TransferableBlock _upstreamErrorBlock;
 
   public FilterOperator(MultiStageOperator upstreamOperator, DataSchema dataSchema, RexExpression filter,
-      long requestId, int stageId, VirtualServerAddress serverAddress) {
-    super(requestId, stageId, serverAddress);
+      OperatorExecutionContext context) {
+    super(context);
```

Comment:
why are some operators calling super(context) and some calling the other API? let's make them consistent.